runtime/src/mips.rs

```diff
@@ -127,7 +141,7 @@ decl_storage! {
         pub Voting get(voting): map T::Hash => Option<Votes<T::AccountId, BalanceOf<T>>>;
 
         /// Active referendums.
-        pub ReferendumMetadata get(referendum_meta): Vec<(MipsIndex, T::Hash)>;
+        pub ReferendumMetadata get(referendum_meta): Vec<(MipsIndex, MipsPriority, T::Hash)>;
```

Comment:
Shouldn't is better to have a struct to store the metadata?

Reply:
I would like a struct because you can put names into each member, which is more descriptive.
About performance in storage, both approaches are the same.

Reply:
Added struct `ReferendumInfo` for capturing attributes of a referendum instead of a tuple.